### PR TITLE
Added a chemical that makes an entity sentient and a ghost role

### PR DESCRIPTION
**Pull request by areitpog**
## About the PR
Adds a chemical which makes an entity sentient; it also has a very complicated recipe

**Screenshots**
https://user-images.githubusercontent.com/77842470/164220889-bcec129c-d72d-4dde-bc5f-14a71a5f08c1.mp4

**Changelog**

:cl: areit
- add: Added a chemical which is able to turn an entity sentient!

### CODE REVIEW

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 1]
```diff
@@ -310,3 +310,21 @@
...
+    Tricordrazine:
+      amount: 1
+    MindbreakerToxin:
+      amount: 1
+    Theobromine:
+      amount: 1
+    Spaceacillin:
+      amount: 1
+    Clonexadone:
+      amount: 1
```
Comment (mirrorcult) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml:328]: this need to be harder to make in the sense that its ingredients are hard to acquire, preferably requiring some chemicals that can only be acquired in salvage (or nocturine?), not just by having a lot of ingredients

Comment (mirrorcult) [reply Resources/Prototypes/Recipes/Reactions/medicine.yml:328]: if you dont want to do that in this PR then imo the recipe should be removed until that can be added

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by UKNOWH** [comment]
looks good , maybe some round I can use it to field a massive monkey army.

**Comment by metalgearsloth** [comment]
Ping for re-open when you get a chance to work on it.

**Pull request closed without merging**